app/controllers/posts_controller.rb

```diff
@@ -790,20 +790,18 @@ def user_posts(guardian, user_id, opts)
   end
 
   def create_params
-    permitted = [
-      :raw,
-      :topic_id,
-      :archetype,
-      :category,
-      # TODO remove together with 'targetUsername' deprecations
-      :target_usernames,
-      :target_recipients,
-      :reply_to_post_number,
-      :auto_track,
-      :typing_duration_msecs,
-      :composer_open_duration_msecs,
-      :visible,
-      :draft_key,
+    permitted = %i[
```

Comment:
After the removal of the inline comment, syntax_tree wants this to be a percent literal. The actual change is deleting of two lines, the comment and the `target_usernames` key.